app/dashboard/templates/shared/menu.html

```diff
@@ -0,0 +1,307 @@
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'grants:new' %}">{% trans "Create Grant" %}</a>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'grants:quickstart' %}">{% trans "About Grants" %}</a>
+              </li>
+            </ul>
+            <ul class="col-md-4 col-lg-2 list-unstyled">
+              <li>
+                <a class="text-reset" href="https://codefund.app/">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/codefund_symbol_dark.svg" %}" alt="Bounties" height="48">
+                  <b class="gc-megamenu-title">{% trans "Codefund" %}</b>
+                </a>
+                <p class="px-2 m-0 mb-2 text-wrap font-smaller-2">{% trans "Ethical ads to power open source" %}</p>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="https://codefund.io/advertisers">{% trans "Advertise with us" %}</a>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="https://codefund.io/publishers">{% trans "Become a publisher" %}</a>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="https://codefund.io">{% trans "About Codefund" %}</a>
+              </li>
+            </ul>
+
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/quests' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url 'quests_index' %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/quests_symbol_dark.svg" %}" alt="Bounties" height="48">
+                  <b class="gc-megamenu-title">{% trans "Quests" %}</b>
+                </a>
+                <p class="px-2 m-0 mb-2 text-wrap font-smaller-2">{% trans "Learn about web3 & earn rewards" %}</p>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'quests_index' %}">{% trans "Explore Quests" %}</a>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'newquest' %}">{% trans "Create Quests" %}</a>
+              </li>
+            </ul>
+
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/kudos' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url 'kudos_main' %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/kudos_symbol_dark.svg" %}" alt="Bounties" height="48">
+                  <b class="gc-megamenu-title">{% trans "Kudos" %}</b>
+                </a>
+                <p class="px-2 m-0 mb-2 text-wrap font-smaller-2">{% trans "Show appreciation for each other." %}</p>
```

Comment:
None of the other descriptions have punctuation, so I think we should remove the `.` here